ddtrace/llmobs/_evaluators/runner.py

```diff
@@ -66,12 +64,14 @@ def start(self, *args, **kwargs):
             return
         super(EvaluatorRunner, self).start()
         logger.debug("started %r to %r", self.__class__.__name__)
-        atexit.register(self.on_shutdown)
```

Comment:
I actually don't know if registering a shutdown function on exit for the evaluator runner does anything; the relevant shutdown behavior is all enforced in _llmobs_ service